### PR TITLE
Update learn menu and footer copy

### PR DESCRIPTION
**Pull request by faihegberg**
This PR adjusts the content under the FAQ section of the mega-menu/footer. Specifically,
- Replace three shuffled FAQ questions with a paragraph about the learn section.
- Replace the "Vaccine FAQs" button with a "Learn more" button which directs the user to the "Learn" page.
- Clean up no longer needed code.

Note: Added comments throughout the PR for clarity.

### CODE REVIEW

File: src/assets/theme/megaMenu.ts [hunk 1]
```diff
@@ -34,5 +34,5 @@ export const megaMenuFooter: MenuTheme = {
   buttonContentHover: COLOR_MAP.NEW_BLUE.PURPLE,
   buttonBorder: COLOR_MAP.GREY_3,
   aboutUsContentAlignment: 'center',
-  aboutUsTextAlignment: 'center',
+  paragraphAlignment: 'center',
```
Comment (faihegberg) [inline src/assets/theme/megaMenu.ts:37]: Renaming `aboutUsTextAlignment` to `paragraphAlignment` as this theme is no longer used for just the "About us" part of the mega-menu/footer.

File: src/cms-content/footer/footer.json [hunk 2]
```diff
@@ -1,19 +1,6 @@
...
-    {
-      "cta": "Can you transmit COVID after vaccination?",
-      "url": "/faq#can-you-transmit-covid-after-vaccination"
-    },
-    {
-      "url": "faq#can-you-get-covid-after-vaccination",
-      "cta": "Can you get COVID even after being vaccinated?"
-    }
-  ],
+  "learn": "Find simple definitions of COVID-related terms, deep dives into COVID science and treatments, and concise answers to pressing COVID questions.",
```
Comment (faihegberg) [inline src/cms-content/footer/footer.json:3]: Discussed with the team this morning whether adjusting the structure of the json here would not cause conflicts with Netlify. Consensus is it _should_ be fine.

File: src/cms-content/footer/index.ts [hunk 3]
```diff
@@ -21,17 +15,11 @@ export interface FeaturedItem {
 }
 
 interface MenuContent {
-  learnLinks: LinkItem[];
+  learn: Markdown;
```
Comment (faihegberg) [inline src/cms-content/footer/index.ts:18]: The section under FAQ is no longer a collection of links, but markdown.

File: src/components/MenuContent/AboutUsSection.tsx [hunk 4]
```diff
@@ -30,7 +30,7 @@ const AboutUsSection: React.FC<{
       ) : (
         <SectionHeader>About Us</SectionHeader>
       )}
-      <AboutCopy>{aboutUsCopy}</AboutCopy>
+      <ParagraphCopy>{aboutUsCopy}</ParagraphCopy>
```
Comment (faihegberg) [inline src/components/MenuContent/AboutUsSection.tsx:33]: Renaming to `paragraphCopy` as this styled component is no longer used for just the "About us" part of the mega-menu/footer.

**Review by mimecuvalo — APPROVED** [review]
LGTM!

**Pull request merged**